.gitignore

```diff
@@ -35,9 +35,16 @@ TAGS
 # C-Lion / IDEA files
 .idea/**
 
-# Python bytecodes
-__pycache__
+# Python
 *.pyc
+*.egg/
+*.eggs/
+*.egg-info/
+.cache/
+.mypy_cache/
+__pycache__/
+build/
```

Comment:
We have a build directory that contains source files.

Reply:
fixed